### PR TITLE
Add post for jabcon 2024

### PR DESCRIPTION
**Pull request by Siedlerchr**

### CODE REVIEW

File: _posts/2024-08-21-AI-chatting.md [hunk 1]
```diff
@@ -59,16 +59,16 @@ JabRef mainly makes use of Large Language Models (LLMs).
 
 Let's break down the dataflow:
 
-* **Create Embeddings:** First of all, JabRef uses a smaller "embedding model" locally on your computer. This model converts the extracted chunks of text from PDFs into numerical representations called embeddings. Embeddings capture the meaning and relationships between words in a way that Large Language Models can understand.
```
Comment (InAnYan) [inline _posts/2024-08-21-AI-chatting.md:62]: Oliver would say: make this a separate PR :)

**Comment by koppor** [comment]
Merged as September, 12th 🤷‍♂️

**Comment by Siedlerchr** [comment]
Then go ahead and rename it
